vignettes/opengraph.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Open Graph"
+description: Customizing social media card previews for pkgdown websites.
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Open Graph}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+[ogp]: https://ogp.me/
+[twitter-validator]: https://cards-dev.twitter.com/validator
+[ogp-validator]: https://search.google.com/structured-data/testing-tool
+[twitter-card]: https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/abouts-cards
+
+Package authors can customize the metadata used by Twitter and the
+[Open Graph protocol][ogp] for rich social media cards.
+In addition to specifying an alternate description for the package
+and any individual articles,
+you may also choose the preview image shown 
+and the style of card used on Twitter.
+
+You can preview and validate the appearance of the social media card previews with 
+
+- [Twitter Card Validator][twitter-validator]
+
+- [Google Structured Data Testing Tool][ogp-validator]
+
+## Site-wide customizations
+
+Metadata for the entire pkgdown website can be specified 
+in the site's `_pkgdown.yml` configuration file in the 
+`home` and `template: opengraph` sections.
+
+```yaml
+home:
+  title: An R package for pool-noodle discovery
+  description: Discover and add pool-noodles to your growing collection.
+  
+template:
+  opengraph:
+    image: 
+      src: man/figures/card.png
+      alt: "Pool noodles configured to form the word poolnoodlr"
+    twitter: 
+      creator: "@hadleywickham"
+      site: "@rstudio"
+      card: summary_large_image
+```
+
+The `home: title` and `home: description` fields override 
+the `Title` and `Description` fields in the package `DESCRIPTION`.
+As mentioned in the `build_home()` documentation,
+it's good practice to change these fields to make your package documentation
+easier to find via search engines.
+
+The `template: opengraph` section allows you to further customize the social media card preview.
+
+By default, pkgdown uses the package's logo for the card preview (if one exists).
```

Comment:
I think this would be a bit easier to follow if you organised in a bulleted list with the same structure all the yaml.

Reply:
I agree and restructured this section into a bulleted list. Happy to iterate again on this if you have feedback.